deployer/templates/hawkular-cassandra-node-dynamic-pv.yaml

```diff
@@ -0,0 +1,114 @@
+id: hawkular-cassandra-node-dynamic-pv
+kind: Template
+apiVersion: v1
+name: 'Hawkular Cassandra Node Template: Persistent Storage'
```

Comment:
I'm missing something then.  I thought all resources had a 'name' and not an 'id'